audit-test-tools/src/lambdas/writeTestDataToAthenaBucket/handler.ts

```diff
@@ -0,0 +1,45 @@
+import { SQSEvent } from 'aws-lambda'
```

Comment:
I think there's a second bit this lambda has to do, which is to trigger the query-completed-queue in the audit account. Except if there's something else that already does that.

Reply:
Actually you're right! Totally forgot about. Will sort that out now